extensions/vendor/functions_arithmetic.yaml

```diff
@@ -0,0 +1,1497 @@
+        return: fp32
+      - args:
+          - name: x
+            value: fp64
+        return: fp64
+  -
+    name: "factorial"
+    description: >
+      Return the factorial of a given integer input.
+      The factorial of 0! is 1 by convention.
+      Negative inputs will raise an error.
+    impls:
+      - args:
+          - value: i32
+            name: "n"
+        options:
+          overflow:
+            values: [ SILENT, SATURATE, ERROR ]
+        return: i32
+      - args:
+          - value: i64
+            name: "n"
+        options:
+          overflow:
+            values: [ SILENT, SATURATE, ERROR ]
+        return: i64
+  -
+    name: "bitwise_not"
+    description: >
+      Return the bitwise NOT result for one integer input.
+    impls:
+      - args:
+          - name: x
+            value: i8
+        return: i8
+      - args:
+          - name: x
+            value: i16
+        return: i16
+      - args:
+          - name: x
+            value: i32
+        return: i32
+      - args:
+          - name: x
+            value: i64
+        return: i64
+  -
+    name: "bitwise_and"
+    description: >
+      Return the bitwise AND result for two integer inputs.
+    impls:
+      - args:
+          - name: x
+            value: i8
+          - name: y
+            value: i8
+        return: i8
+      - args:
+          - name: x
+            value: i16
+          - name: y
+            value: i16
+        return: i16
+      - args:
+          - name: x
+            value: i32
+          - name: y
+            value: i32
+        return: i32
+      - args:
+          - name: x
+            value: i64
+          - name: y
+            value: i64
+        return: i64
+  -
+    name: "bitwise_or"
+    description: >
+      Return the bitwise OR result for two given integer inputs.
+    impls:
+      - args:
+          - name: x
+            value: i8
+          - name: y
+            value: i8
+        return: i8
+      - args:
+          - name: x
+            value: i16
+          - name: y
+            value: i16
+        return: i16
+      - args:
+          - name: x
+            value: i32
+          - name: y
+            value: i32
+        return: i32
+      - args:
+          - name: x
+            value: i64
+          - name: y
+            value: i64
+        return: i64
+  -
+    name: "bitwise_xor"
+    description: >
+      Return the bitwise XOR result for two integer inputs.
+    impls:
+      - args:
+          - name: x
+            value: i8
+          - name: y
+            value: i8
+        return: i8
+      - args:
+          - name: x
+            value: i16
+          - name: y
+            value: i16
+        return: i16
+      - args:
+          - name: x
+            value: i32
+          - name: y
+            value: i32
+        return: i32
+      - args:
+          - name: x
+            value: i64
+          - name: y
+            value: i64
+        return: i64
+
+aggregate_functions:
```

Comment:
Are `count` and `approx_count_distinct` implemented already?

Reply:
All I did was copy the yaml files from `substrait-io/substrait` here because `go get` doesn't resolve submodules. 

So if those are expected to exist but aren't here, that would mean they are missing from the yaml files in the substrait main repo.

Reply:
Maybe some files didn't get copied? Like these:
https://github.com/substrait-io/substrait/blob/main/extensions/functions_aggregate_approx.yaml
https://github.com/substrait-io/substrait/blob/main/extensions/functions_aggregate_generic.yaml

Reply:
ah must have missed those! thanks. i'll add those and update

Reply:
just pushed the files